metis-normalization/src/main/java/eu/europeana/normalization/normalizers/DatesNormalizer.java

```diff
@@ -286,6 +288,7 @@ private DateNormalizationResult normalizeProperty(
   private void noMatchIfValidAndTimeOnly(DateNormalizationResult dateNormalizationResult) {
     if (dateNormalizationResult.getDateNormalizationExtractorMatchId() != DateNormalizationExtractorMatchId.INVALID
         && dateNormalizationResult.getEdtfDate().isTimeOnly()) {
+
       // TODO: 21/07/2022 In the result only the match id is declared NO_MATCH but the contents are still present in the object. Is that okay?
```

Comment:
This is okay, but as before we don't expect to have just the time and not the date.
To be considered on how to structure this, if at all to be changed.